src/app/components/icons/index.tsx

```diff
@@ -11,7 +12,13 @@ export const Ellipsis = () => (
     <path d="M1.6 18.8h5.8v-5.6H1.6v5.6zm11.5 0h5.8v-5.6h-5.8v5.6zm11.5 0h5.8v-5.6h-5.8v5.6z" />
   </svg>
 );
-export const LeftChevron = ({ className }: { className?: string }) => (
+
+type ChevronProps = {
+  className?: string;
+  dir?: Direction;
+};
+
+export const LeftChevron = ({ className, dir = 'ltr' }: ChevronProps) => (
```

Comment:
I'm a bit confused on this, why do we need to pass the direction here? The left chevron should always be pointing left and the right chevron should be always be right? This looks weird to me for when dir is rtl 
![image](https://github.com/user-attachments/assets/04d7645a-9e98-49c2-8ad5-af88400c0cf2)